packages/loader/container-definitions/src/fluidPackage.ts

```diff
@@ -3,82 +3,72 @@
  * Licensed under the MIT License.
  */
 
-/**
- * Person definition in a npm script
- */
-export interface IPerson {
-    name: string;
-    email: string;
-    url: string;
-}
+ /**
+  * Specifies an environment on Fluid property of a IFluidPackage
+  */
+export interface IFluidPackageEnvironment {
 
-/**
- * Typescript interface definition for fields within a npm module's package.json.
- */
-export interface IPackage {
-    // General access for extended fields
-    [key: string]: any;
-    name: string;
-    version: string;
-    description?: string;
-    keywords?: string[];
-    homepage?: string;
-    bugs?: { url: string; email: string };
-    license?: string;
-    author?: IPerson;
-    contributors?: IPerson[];
-    files?: string[];
-    main?: string;
-    // Same as main but for browser based clients (check if webpack supports this)
-    browser?: string;
-    bin?: { [key: string]: string };
-    man?: string | string[];
-    repository?: string | { type: string; url: string };
-    scripts?: { [key: string]: string };
-    config?: { [key: string]: string };
-    dependencies?: { [key: string]: string };
-    devDependencies?: { [key: string]: string };
-    peerDependencies?: { [key: string]: string };
-    bundledDependencies?: { [key: string]: string };
-    optionalDependencies?: { [key: string]: string };
-    engines?: { node: string; npm: string };
-    os?: string[];
-    cpu?: string[];
-    private?: boolean;
+    /**
+     * The name of the target. For a browser environment, this could be umd for scripts
+     * or css for styles.
+     */
+    [target: string]: undefined | {
```

Comment:
Do we want to more strongly type the target to object mapping? Worried about it being easy to forget to include library for JS

Reply:
the IFluidBrowserPackage adds this typing